src/utils/get.js

```diff
@@ -0,0 +1,12 @@
+const get = (object, path, defaultVal) => {
```

Comment:
I think my approach is better in term of performance :zap: 

 - Because you split `path` for every step
 - Because this is recursion vs iteration